plugins/woocommerce-admin/client/products/edit-product-page.tsx

```diff
@@ -109,7 +130,14 @@ const EditProductPage: React.FC = () => {
 						</div>
 					</ProductFormLayout>
 				) }
-			{ product &&
+			{ productVariation && product && (
```

Comment:
Note that this doesn't handle 404 cases where a variation empty state does not yet exist, but this also isn't yet handled on for products, so we might want to follow-up on this.